encord/orm/label_log.py

```diff
@@ -9,7 +9,8 @@
 class LabelLog:
     log_hash: str
     user_hash: str
-    annotation_hash: str
```

Comment:
@denis-cord, are we sure people aren't using this field? This looks like a break in backwards compatibility for no a good reason.

Why not keep both, at least temporarily?